src/main/java/org/reaktivity/nukleus/maven/plugin/internal/generate/EnumFlyweightGenerator.java

```diff
@@ -43,13 +44,15 @@
     public EnumFlyweightGenerator(
         ClassName enumName,
         ClassName flyweightName,
-        ClassName enumTypeName)
+        ClassName enumTypeName,
+        AstType valueType)
```

Comment:
Just realized that we don't typically use `AstType` directly in generator code, usually resolve to `TypeName` first, so please do the same for `valueType` and rename to `valueTypeName`.

In this case, it will likely be passed in as `TypeName.INT` for now, and something more complex like `ClassName` later when we further enhance `enum`.